docs/getting_started.md

```diff
@@ -4,15 +4,15 @@ So you've got Shadow installed and your machine configured. Its time to see what
 
 When installing Shadow, the main executable was placed in `/bin` in your install prefix (`~/.shadow/bin` by default). As a reminder, it would be helpful if this location was included in your environment `PATH`.
 
-`shadow` is the main Shadow binary executable. It contains most of the simulator's code, including events and the event engine, the network stack, and the routing logic.
+`shadow` is the main Shadow binary executable. It contains most of the simulator's code, including events and the event engine, the network stack, and the routing logic. Shadow's event engine supports multi-threading using the `-p` or `--parallelism` flags (or their corresponding [configuration file option](shadow_config.md#generalparallelism)) to simulate multiple hosts in parallel.
 
-The `shadow` binary is capable of appending custom **function interposition** libraries to the `LD_PRELOAD`  environment variable to make it possible to intercept real operating system functions and manage them in the simulation environment. The `shadow` binary also assists with running `valgrind`, mostly for debugging and development purposes. For more information:
+The `shadow` binary is capable of appending custom **function interposition** libraries to the `LD_PRELOAD` environment variable to make it possible to intercept real operating system functions and manage them in the simulation environment. The `shadow` binary also assists with running `valgrind`, mostly for debugging and development purposes. For more information:
 
 ```bash
 shadow --help
 ```
 
-## Shadow plug-ins
+## Supported applications
```

Comment:
Maybe worth a note for 1.x users that plugins are no longer needed? e.g. "Since 2.0, applications can typically be run under Shadow without modification. Currently the only requirements are..."

Reply:
Added